### ISSUE TITLE
Failed to build Havit.Blazor locally

### ISSUE THREAD

**Issue by sshulik** [havit/Havit.Blazor#33]
I want to propose a few PRs with new components and improvements but can't build the Havit solution locally. My problems:

1. During the build I get the next error:
![image](https://user-images.githubusercontent.com/2964570/139653977-95a8405c-871d-4cd8-a7f5-9af7e95f5b9b.png)

Looks like it's some CI artifacts, but I am pretty sure that it's shouldn't be in common code. Is it possible to move it to CI processes or provide a solution on how to disable it locally?

2. Problem with `BootstrapIcon`. In remarks for I found the solution how to solve it:
```
	/// - Update wwwroot\fonts\bootstrap-icons.woff
	/// - Update wwwroot\fonts\bootstrap-icons.woff2
	/// - Update wwwroot\bootstrap-icons.css
	/// - Update bootstrap-icons.json in this folder.
```
but looks like it does not work for me. Is it possible to provide Powershell (and bash for macOS/Linux) to trigger update these files to automate regeneration of these properties?

**Comment by hakenr** [comment]
There is a SourceLink issue related to the fact we are deploying the project from Azure DevOps. Making the SourceLink conditional should resolve the issue. Will push the fix today. 

I believe the fix will also resolve your issue 2).

**Comment by hakenr** [comment]
@sshulik, pull the latest version from master and **restart Visual Studio**. The build should succeed now.

(fixed with 445c740483703b1040793743b0ddb70116f477f9)
